heka/sandbox/filters/firefox_weekly_dashboard.lua

```diff
@@ -81,7 +81,7 @@ local function update_week(ts, cid, day)
         error("data is in the past, this report doesn't back fill")
     end
 
-    local country = fx.get_country_id(read_message("Fields[geo]"))
+    local country = fx.get_country_id(read_message("Fields[country]"))
```

Comment:
Do we need to `s/geo/country/` on line 121 too? or is it called "geo" for compatibility?